tests/unit/CardUtilsTest.ts

```diff
@@ -215,4 +215,41 @@ describe('CardUtils', () => {
             expect(feedName).toBe(undefined);
         });
     });
+
+    describe('maskCardNumber', () => {
```

Comment:
Can you add a test for the card name with a string?

Reply:
Do you mean empty string? We have it [here](https://github.com/Expensify/App/pull/52578/files#diff-5d07e5479e345b961658f705e6c8710f6cc4df22e642d466ddca3ebb15c77d83R250-R253)

Reply:
Covered by this one `CREDIT CARD...6607`, thanks!